diff --git a/include/configs/rpi.h b/include/configs/rpi.h
index 8e56bdc84a8..4eb923f8619 100644
--- a/include/configs/rpi.h
+++ b/include/configs/rpi.h
@@ -20,6 +20,9 @@
 	(&((struct bcm2835_timer_regs *)BCM2835_TIMER_PHYSADDR)->clo)
 #endif
 
+#define CONFIG_EXTRA_ENV_SETTINGS \
+    "bootdelay=-2\0"
+
 /* Memory layout */
 #define CFG_SYS_SDRAM_BASE		0x00000000
 #define CFG_SYS_UBOOT_BASE		CONFIG_TEXT_BASE
